pkg/crds/yaml/generated/management.cattle.io_clusterproxyconfigs.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.0
+  name: clusterproxyconfigs.management.cattle.io
+spec:
+  group: management.cattle.io
+  names:
+    kind: ClusterProxyConfig
+    listKind: ClusterProxyConfigList
+    plural: clusterproxyconfigs
+    singular: clusterproxyconfig
+  scope: Namespaced
+  versions:
+  - name: v3
+    schema:
+      openAPIV3Schema:
+        description: ClusterProxyConfig determines which downstream requests will
+          be proxied to the downstream cluster for requests that contain service account
+          tokens.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          enabled:
+            description: Enabled indicates whether downstream proxy requests for service
+              account tokens is enabled. Enabled
```

I suppose this was a leftover.

```suggestion
              account tokens is enabled.
```

---

I removed the extra Enabled and regenerated